x-pack/plugins/observability/public/components/shared/field_value_suggestions/index.tsx

```diff
@@ -7,7 +7,7 @@
 
 import React, { useState } from 'react';
 
-import { useDebounce } from 'react-use';
+import useDebounce from 'react-use/lib/useDebounce';
```

Comment:
This PR fixes an eslint rule that should have been firing, but wasn't. I've updated this import to comply with the fixed rule.